### PR TITLE
DP-24527: Problem with link in Service Page banner not formating correctly

### PR DESCRIPTION
**Pull request by arthurbaghdas**
**Description:**
Explain the technical implementation of the work done.

**Jira:** (Skip unless you are MA staff)
DP-24527

**To Test:**
- [ ] Add steps to test this feature

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

File: changelogs/DP-24527.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Changed:
+#  - description: Automating the release branch.
+#    issue: DP-10166
+#  - description: Second change item that needs a description.
+#    issue: DP-19875
+#  - description: Third change item that needs a description along with an issue.
+#    issue: DP-19843
+#
+Fixed:
+  - description: Describe the change. If you need multiple lines, start the first line with the following "|-" characters.
```
Comment (deviantintegral) [inline changelogs/DP-24527.yml]: Looks like the description is missing.

File: docroot/themes/custom/mass_theme/templates/content/node--service-page.html.twig [hunk 2]
```diff
@@ -70,7 +70,7 @@
   {% if node.field_log_in_links is not empty %}
     {% for item in node.field_log_in_links %}
       {% set button = {
-          "href": item.uri,
+          "href": item.url,
```
Comment (deviantintegral) [inline docroot/themes/custom/mass_theme/templates/content/node--service-page.html.twig:73]: One character makes all the difference!

**Review by deviantintegral — COMMENTED** [review]
For sure the changelog text needs fixing, but the code change looks good.

Given this bug made it to production, should we write a functional test too?

File: changelogs/DP-24527.yml (hunk 1 above)
Comment (arthurbaghdas) [reply changelogs/DP-24527.yml]: Thank you, missed that info. I don't think we need a specific test for this case.

**Review by deviantintegral — APPROVED** [review]

**Pull request merged**